custom/opm/reports.py

```diff
@@ -888,7 +896,7 @@ def row_objects(self):
         self._debug_data = []
         for index, row in enumerate(self.get_rows(self.datespan), 1):
             try:
-                rows.append(self.get_row_data(row, index=index))
+                rows.append(self.get_row_data(row, index=1))
```

Comment:
is this supposed to be `row.serial_number`? otherwise i'm not sure what the point of setting them above is.